SlicerRadiomicsCLI/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+
+#-----------------------------------------------------------------------------
+set(MODULE_NAME SlicerRadiomicsCLI)
+
+#-----------------------------------------------------------------------------
+
+#
+# SlicerExecutionModel
+#
+find_package(SlicerExecutionModel REQUIRED)
+include(${SlicerExecutionModel_USE_FILE})
+
+#
+# ITK
+#
+set(${PROJECT_NAME}_ITK_COMPONENTS
+  ITKIOImageBase
+  ITKSmoothing
+  )
+find_package(ITK 4.6 COMPONENTS ${${PROJECT_NAME}_ITK_COMPONENTS} REQUIRED)
+set(ITK_NO_IO_FACTORY_REGISTER_MANAGER 1) # See Libs/ITKFactoryRegistration/CMakeLists.txt
+include(${ITK_USE_FILE})
+
+#-----------------------------------------------------------------------------
+set(MODULE_INCLUDE_DIRECTORIES
+  )
+
+set(MODULE_SRCS
+  )
+
+set(MODULE_TARGET_LIBRARIES
+  ${ITK_LIBRARIES}
+  )
+
+#-----------------------------------------------------------------------------
+SEMMacroBuildCLI(
+  NAME ${MODULE_NAME}
+  TARGET_LIBRARIES ${MODULE_TARGET_LIBRARIES}
+  INCLUDE_DIRECTORIES ${MODULE_INCLUDE_DIRECTORIES}
+  ADDITIONAL_SRCS ${MODULE_SRCS}
+  )
```

Comment:
Did you try to configure the project ? It looks like `${MODULE_NAME}.cxx` is missing

Reply:
I guess, we could look into factoring out from `SEMMacroBuildCLI` macro the part installing the executable files and xml. That way it would ensure it works on all platform and files are packaged in the right location